homeworks/Makc.Sas_MakcSas/homework_3/homework_3_2.js

```diff
@@ -0,0 +1,16 @@
+let startDate = 1;
+function powerTwo(value) {
+    if (startDate === value) {
+        return true;
+    }
+    if (startDate > value) {
+        return false;
+    }
+    startDate *= 2;
+    return powerTwo(value);
```

Comment:
in general, you do not need `startDate` variable in this script, rethink your approach